go/interpreter/lfvm/interpreter_test.go

```diff
@@ -310,29 +310,42 @@ func TestInstructionsGasConsumption(t *testing.T) {
 	}
 }
 
-func TestRunReturnsEmptyResultOnEmptyCode(t *testing.T) {
-	// Get tosca.Parameters
-	params := tosca.Parameters{
-		Input:  []byte{},
-		Static: true,
-		Gas:    10,
-	}
-	code := make([]Instruction, 0)
+func TestInterpreter_ExecutionTerminationCode(t *testing.T) {
```

Comment:
We test a property formulated as a sentence, with a verb:
```suggestion
func TestInterpreter_ExecutionTerminates(t *testing.T) {
```